DeveloperGame/DeveloperGame/ComputerGeekGame.cs

```diff
@@ -25,8 +45,37 @@ public void Run()
             {
                 PrintLeaderBoard();
             }
-           
-        } while (!UserChoseToQuit(userMenuChoice));
+            if (UserChoseToQuit(userMenuChoice))
+            {
+                return;
+            }
+        }
+    }
+
+    private void ReadScoresFromFile()
+    {
+        StreamReader reader = File.OpenText(_scoreBoardfile);
+
+        string line = reader.ReadLine();
+
+        while (line != null)
+        {
+            string[] scoreAndName = line.Split(' ');
+
+            if (scoreAndName.Length == 2)
```

Comment:
Won't you have 3 things? Name, Score and Time?

Reply:
Yes, I did not really think through how I am going to get the time in there.

Reply:
Fixed.